tests/conftest.py

```diff
@@ -331,3 +332,18 @@ def parametrize_zlib_backend(
     yield
 
     set_zlib_backend(original_backend)
+
+
+@pytest.fixture()
+def cleanup_payload_pending_file_closes(
```

Comment:
Would this work as an async function and just await on the futures?

Reply:
https://github.com/aio-libs/aiohttp/pull/10943